<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EEDSmoothingDialog</class>
 <widget class="QDialog" name="EEDSmoothingDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1146</width>
    <height>1000</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>EED Smoothing</string>
  </property>
  <widget class="QLabel" name="label_EEDSmoothing_ImgDisplay">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>40</y>
     <width>511</width>
     <height>491</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_EEDSmoothing_inputs">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>30</y>
     <width>371</width>
     <height>301</height>
    </rect>
   </property>
   <property name="title">
    <string>Inputs</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label__EEDSmoothing_timeStep">
          <property name="text">
           <string>Step Size</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_EEDSmoothing_timeStep"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="label__EEDSmoothing_iterNum">
          <property name="text">
           <string>Iteration Number</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_EEDSmoothing_iterNum"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="label__EEDSmoothing_contPar">
          <property name="text">
           <string>Contrast Parameter</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_EEDSmoothing_contPar"/>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton__EEDSmoothing_LoadImgData">
        <property name="text">
         <string>Load Data</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioButton_EEDSmoothing_explScheme">
        <property name="text">
         <string>Explicit Scheme</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioButton_EEDSmoothing_FED">
        <property name="text">
         <string>Fast Explicit Scheme(FED)</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QLabel" name="label_EEDSmoothing_FEDInnerCycles">
          <property name="text">
           <string>Number of Inner Cycles</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_EEDSmoothing_FEDInnerCycles"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <widget class="QLabel" name="label_EEDSmoothing_totTime">
          <property name="text">
           <string>Total Stopping Time</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_EEDSmoothing_TotTime"/>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_EEDSmoothing_run">
      <property name="text">
       <string>Run</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>860</y>
     <width>451</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label_EEDSmoothing_saveImName">
      <property name="text">
       <string>Name</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit__EEDSmoothing_saveImgName"/>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_EEDSmoothing_svaeImgName">
      <property name="text">
       <string>Save</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
